src/shared/sam/cli/samCliBuild.ts

```diff
@@ -14,35 +14,39 @@ export interface SamCliBuildInvocationArguments {
     /**
      * The path to a folder where the built artifacts are stored.
      */
-    buildDir: string,
+    buildDir: string
     /**
      * Resolves relative paths to the function's source code with respect to this folder.
      * If omitted, relative paths are resolved with respect to the template's location.
      */
-    baseDir?: string,
+    baseDir?: string
     /**
      * Location of the SAM Template to build
      */
-    templatePath: string,
+    templatePath: string
     /**
      * Manages the sam cli execution.
      */
-    invoker: SamCliProcessInvoker,
+    invoker: SamCliProcessInvoker
     /**
      * If your functions depend on packages that have natively compiled dependencies,
      * use this flag to build your function inside an AWS Lambda-like Docker container.
      */
-    useContainer?: boolean,
+    useContainer?: boolean
     /**
      * Specifies the name or id of an existing Docker network to Lambda Docker containers should connect to,
      * along with the default bridge network.
      * If not specified, the Lambda containers will only connect to the default bridge Docker network.
      */
-    dockerNetwork?: string,
+    dockerNetwork?: string
     /**
      * Specifies whether the command should skip pulling down the latest Docker image for Lambda runtime.
      */
-    skipPullImage?: boolean,
+    skipPullImage?: boolean
+    /**
+     * -m or --manifest for SAM CLI
```

Comment:
The other field comments have been heavily influenced by the sam cli documentation. Please replace with:

The path to a custom dependency manifest (ex: package.json) to use instead of the default one.

Reply:
✔️